automl/automlVisionDataset.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*eslint no-console: 0*/
+
+/**
+ * This application demonstrates how to perform basic operations on dataset
+ * with the Google AutoML Vision API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/vision/automl/docs/
+ */
+
+`use strict`;
+
+function createDataset(projectId, computeRegion, datasetName, multilabel) {
+  // [START automl_vision_createDataset]

```

Comment:
Please fix **all region tags** – this should be `automl_vision_create_dataset` ([canonical](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/vision/automl/automl_vision_dataset.py#L30)) – we don't camelCaseAnyRegionTagsAfaik :)